### PR TITLE
dev/refactor: Refactor Code & Add Receipt

### PR DESCRIPTION
**Pull request by sgoudham**

### CODE REVIEW

File: src/main/java/org/somerville/swag/data/entity/Order.java [hunk 1]
```diff
@@ -60,6 +60,22 @@ public BigDecimal getTotal() {
 
     public String getFormattedTotal() { return "£" + getTotal().setScale(2, RoundingMode.HALF_UP); }
 
+    public String getReceipt(String forename) {
+        StringBuilder receipt = new StringBuilder("Thanks For Ordering " + forename + "! Your Swag Will Be With You Soon :D\n\nPurchased Swag:");
+        for (OrderLine orderLine : orderLines) {
+            Product product = orderLine.getProduct();
+            receipt.append("\n")
```
Comment (AlanRobinson1337) [inline src/main/java/org/somerville/swag/data/entity/Order.java:67]: Great String building

**Review by AlanRobinson1337 — APPROVED** [review]
Good refactoring

File: src/main/java/org/somerville/swag/data/entity/Order.java (hunk 1 above)
Comment (sgoudham) [reply src/main/java/org/somerville/swag/data/entity/Order.java:67]: Thank you! I opted for StringBuilder as it's more efficient and doesn't waste memory.

**Review by hzl616 — APPROVED** [review]
The UX is much more personalised. Great finishing touches to the experience

**Pull request merged**